pkg/internal/kube/store.go

```diff
@@ -408,7 +415,7 @@ func (s *Store) ServiceNameNamespaceForIP(ip string) (string, string) {
 
 	name, namespace := "", ""
 	if om, ok := s.objectMetaByIP[ip]; ok {
-		name, namespace = s.serviceNameNamespaceForMetadata(om.Meta)
+		name, namespace = s.serviceNameNamespaceForMetadata(om.Meta, "")
```

Comment:
Hm, so passing "" here will create disparity between reverse IP resolution of names and what we set the service name with the template.

Reply:
Yes, this was actually on purpose here, since a pod is always a single network namespace. I wanted the trace view to have a per container differentiation and didn't care / think about the span properties.

I could imagine to try to map the dst / src port based on the exposed containerPorts - what do you think about this?

Reply:
Currently I think the port is unavailable from the cache service.